elasticsearch/utils/logging

```diff
@@ -80,11 +80,11 @@ check_index_exists() {
   [[ $response_code -eq 200 ]] || exit 1
 }
 
-# Pull in the certs provided in our secret and generate our necessary jks and truststore files
-build_jks_truststores() {
```

Comment:
i think we need to keep this function for now and then we can remove it afterwards once EO is only configuring clusters with .p12